src/JavaSession1/ArrayRepetition.java

```diff
@@ -0,0 +1,59 @@
+package JavaSession1;/*Question 6: There is an array with every element repeated twice except one. Find that element*/
+
+
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Scanner;
+
+public class ArrayRepetition {
+
+    public static void main(String[] args) {
+
+        int[] integerArray;
+        Scanner sc = new Scanner(System.in);
+        System.out.println("Enter the size of Array");
+        int arraySize = sc.nextInt();
+        integerArray = new int[arraySize];
+        insertInArray(integerArray, arraySize);
+        System.out.println("Element Which Is Repeated Once In Array : " + repeatedElementHashMap(integerArray, arraySize));
+
+
+    }
+
+    private static int repeatedElement(int[] integerArray, int arraySize) {
```

Comment:
method name can be improved. It is not defining the purpose correctly. Also, you are passing the arraySize which is not required at all.